# HG changeset patch
# User Rik <rik@octave.org>
# Date 1374174092 25200
#      Thu Jul 18 12:01:32 2013 -0700
# Node ID c6a39f7f193db04bf4d1ef5bf2f44b5026ad082f
# Parent  dd783018709b626611c6359c46166261a0d0cad6
doc: Use '@@' to produce '@' in strings.txi

* doc/interpreter/strings.txi: Use '@@' to produce '@'.

diff --git a/doc/interpreter/strings.txi b/doc/interpreter/strings.txi
--- a/doc/interpreter/strings.txi
+++ b/doc/interpreter/strings.txi
@@ -231,17 +231,17 @@ only of blank characters (ASCII code 32)
 
 Strings can be concatenated using matrix notation
 (@pxref{Strings}, @ref{Character Arrays}) which is often the most natural
 method.  For example:
 
 @example
 @group
 fullname = [fname ".txt"];
-email = ["<" user "@" domain ">"];
+email = ["<" user "@@" domain ">"];
 @end group
 @end example
 
 @noindent
 In each case it is easy to see what the final string will look like.  This
 method is also the most efficient.  When using matrix concatenation the parser
 immediately begins joining the strings without having to process
 the overhead of a function call and the input validation of the associated
